Makefile

```diff
@@ -98,7 +89,6 @@ rebuild-dependencies.tsv: godeps
 # PPA includes the required mongodb-server binaries.
 install-dependencies:
 	@echo Adding juju PPAs for golang and juju
```

Comment:
I think install-dependencies should also do this:

```
        go get -u github.com/rogpeppe/godeps
        $(GOPATH)/bin/godeps -u dependencies.tsv
```

Anyone who is wanting to build Juju is going to need godeps and grab the dependencies so I think the build target should do it for them.